src/tcgwars/logic/impl/gen4/DiamondPearl.groovy

```diff
@@ -1300,7 +1300,10 @@ public enum DiamondPearl implements LogicCardInfo {
             onAttack {
               flip {
                 targeted (defending) {
-                  defending.damage += self.damage
+                  //defending.damage += self.damage
+                  directDamage self.damage, defending
+                  def dc = self.damage / 10
+                  bc "Moved $dc damage counters from $self to $defending."
```

Comment:
I think `directDamage` should already state how much damage it did to the defending Pokemon. Probably better repurposed to mention that it heals Wobbuffet

Reply:
Yeah, it seems to do. I'm rewriting this entirely though, since wobbuffett actually moves the counters (so it doesn't count as healing, checking at some rulings).

Need to check on metal goggles, since it doesn't seem to work with the moving counters way however... (checked on Xerneas GX)